src/components/projects/projectId/attributes/attributeId/ExecutionContainer.tsx

```diff
@@ -56,26 +57,27 @@ export default function ExecutionContainer(props: ExecutionContainerProps) {
             dispatch(setModalStates(ModalEnum.VIEW_RECORD_DETAILS, { record: postProcessRecordByRecordId(res) }));
         });
     }
-
     return (<div>
         <div className="mt-8 text-sm leading-5">
             <div className="text-gray-700 font-medium mr-2">
                 Execution
             </div>
 
             <div className="flex items-center">
-                <div className="text-gray-500 font-normal">You can execute your attribute calculation
+                <div className="text-gray-500 font-normal inline-flex flex-col gap-y-2 w-full">You can execute your attribute calculation
                     on all records, or test-run it on 10 examples (which are sampled randomly). Test results are
                     shown
-                    below after computation.</div>
+                    below after computation.{
+                        props.currentAttribute.dataType == DataTypeEnum.LLM_RESPONSE && <span >Note that LLM results are cached for the same set of settings to ensure that a critical error during the final execution doesn't lose the already calculated values</span>
+                    }</div>
                 {requestedSomething && <div className="inline-block">
                     <LoadingIcon color="indigo" />
                 </div>}
 
                 <Tooltip content={TOOLTIPS_DICT.ATTRIBUTE_CALCULATION.EXECUTE_10_RECORDS} color="invert" placement="bottom" className="ml-auto">
                     <button onClick={calculateUserAttributeSampleRecords}
                         disabled={props.currentAttribute.state == AttributeState.USABLE || props.currentAttribute.state == AttributeState.RUNNING || requestedSomething || props.tokenizationProgress < 1 || props.checkUnsavedChanges}
-                        className={`bg-white text-gray-700 text-xs font-semibold px-4 py-2 rounded-md border border-gray-300 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 disabled:opacity-50 disabled:cursor-not-allowed`}>
+                        className={`bg-white text-gray-700 text-xs font-semibold px-4 py-2 rounded-md border whitespace-nowrap border-gray-300 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 disabled:opacity-50 disabled:cursor-not-allowed`}>
```

Comment:
i know it's a code from before but we can use the classes as text, since we don't have any conditions